effekt/shared/src/main/scala/effekt/core/Transformer.scala

```diff
@@ -87,7 +87,7 @@ object Transformer extends Phase[Typechecked, CoreTransformed] {
           // like in asSeenFrom we need to make up cparams, they cannot occur free in the result type
           val capabilities = effects.canonical
           val tparams = tps.drop(tparamsInterface.size)
-          val bparamsBlocks = bps.map(b => transform(b.tpe))
+          val bparamsBlocks = bps.map(b => transform(b.tpe.get))
```

Comment:
This should be there, because interface declarations are annotated, right? Maybe we can document this here like above